### DISCUSSION TITLE
Enhancing Security: Restricting Administrator Access to User Creation Only

### DISCUSSION THREAD

**Discussion by mauroalexandre** [frangoteam/FUXA#1141] [Ideas]
Currently, administrators cannot have access solely to user creation without having access to the entire editor. In many scenarios, the person responsible for access control is not supposed to manipulate and edit HMI information, among other things.

What would be the best approach to address this issue?

**Comment by Jabbajoe** [comment]
Hello guys, reviving this conversation!

Is there an approach to this problem? Currently, we have some use cases where the end user needs to manage other users (creating, reseting password, changing role, deleting). Today, the only way to do this is giving admin access, then accessing the Editor and managing the users. We need to streamline and facilitate this process, providing a way to manage users without access to the Editor

**Comment by mauroalexandre** [comment]
@unocelli Do you have any suggestions or plans in this regard?

**Comment by unocelli** [comment]
Roles should be defined for the different sections, manage the access to the sections via routing (url) should be simple. the problem will be if we will manage in details, access pro function (write tags/devices, write tag options, ...)
